Backend/dev/migrations/dynamic-offer-driver-app/0121-updated-transporter-config-table.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE atlas_driver_offer_bpp.transporter_config DROP COLUMN ride_travelled_dist_threshold_when_pickup_or_dest_is_diff;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config DROP COLUMN ride_travelled_dist_threshold_when_pickup_and_dest_is_same;
+UPDATE atlas_driver_offer_bpp.transporter_config SET pickup_loc_threshold = 500;
+UPDATE atlas_driver_offer_bpp.transporter_config SET drop_loc_threshold = 500;
+UPDATE atlas_driver_offer_bpp.transporter_config SET ride_time_estimated_threshold = 700;
```

Comment:
Add not null constraints as these values can't be maybe. Add not null on `waitingTimeEstimatedThreshold` too as you have removed maybe from it.